_sidebar.md

```diff
@@ -35,10 +33,19 @@
 * [[Coding Style and Conventions]] ⚖
 * [[Accessibility Guideline]] ♿
 * [[Avoid Roadblocks]] 🚧
+* [[Questions]] ❓
 
 ### Development/Testing
 
 * [[Quick Start]]
 * [[Required Dependencies]] 📌
 * [[Sample Development]] 📱
-* [[Fabric Bot Services]] 🤖
\ No newline at end of file
+* [[Fabric Bot Services]] 🤖
+
+### Project Structure
+
+* [[How the Project is Organized]] 🗄️
```

Comment:
Did you have docs for these too or just place-holders? (I think we should probably put them all together as part of a single PR to address #5 all-up. Do we need more discussion there first?)

I like having the separate section here, though we could probably move the 'Why Microsoft Supports' to the bottom of the 'about' section at the top (and include more details about the .NET Foundation on the page we have for them there).

I think that leaves this section to be around 'How the organization and project is organized' (together or two pages?), 'How to join the Toolkit team?', and 'Hall of Fame'?

Thoughts?

Reply:
Correct, this is going to be all part of #5 and there is still plenty of work that needs to be done as docs are still in the draft phase. 

Yeah, "How the organization and project is organized" is one page, join the Toolkit team, and Hall of Fame will all be part of the same section. 

Thanks for the feedback 😊

Reply:
Cool, do we want to leave these out of this PR for now to get your other fixes in and then start brainstorming in Issue #5 for content for the new pages?